action/protocol/poll/blockmeta.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/golang/protobuf/ptypes"
```

Comment:
ptypes.Timestamp() is also deprecated, replace as well 
```
// Deprecated: Call the ts.AsTime and ts.CheckValid methods instead.
func Timestamp(ts *timestamppb.Timestamp) (time.Time, error) {
```

Reply:
and do this for all other files

Reply:
Sure! I'll replace all deprecated functions which mentioned in this page.
https://pkg.go.dev/github.com/golang/protobuf/ptypes